src/app/layout.tsx

```diff
@@ -29,7 +30,7 @@ export default function RootLayout({
       <body
         className={`${geistSans.variable} ${geistMono.variable} antialiased`}
       >
-        <Provider>{children}</Provider>
+        {children}
```

Comment:
Please align with @duchelle5093 on the role of this Provider you are removing. We shall have a discussion tomorrow on how to handle merge conflicts. I'm on sick leave today. But general rule of thumb, avoid removing people's changes without being 100% sure you're not breaking something